elastic/kibana
# [Infrastructure UI] Fix condition to show k8s dashboard link

## Summary

closes #151273 

This PR fixes a bug when the Kubernetes Dashboards link is displayed. It will only show when Kubernetes Pods is selected

![out](https://user-images.githubusercontent.com/2767137/219019027-2b6624e9-2396-4950-b34e-5090ff170577.gif)

### How to test it
- Connect your local kibana to an oblt cluster
- Go to Observability > Inventory UI
- Change "Show" and select Kubernetes Pods to see the link and any other option to hide it.
